kubernetes/replicator-aks-cc/docs/index.rst

```diff
@@ -36,7 +36,7 @@ The following applications or libraries are required to be installed and availab
 +------------------+----------------+-------------------------------------------------------------------------------------+
 | ``az``           | ``2.10.1``     | https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest  |
 +------------------+----------------+-------------------------------------------------------------------------------------+
-| ``ccloud``       | ``v1.0.0``     | https://docs.confluent.io/ccloud-cli/current/install.html                           |
+| ``ccloud``       | ``v1.24.0``     | https://docs.confluent.io/ccloud-cli/current/install.html                           |
```

Comment:
@rspurgeon this may cause `rst` to complain because of the misaligned `|` -- did you happen to verify the docs build still works?

Reply:
sorry, I did not